### ISSUE TITLE
Error in UseMethod("xml_find_all") 

### ISSUE THREAD

**Issue by SaraMorsy** [carpentries/sandpaper#339]
Hi there,
I hope you are fine. GitHub page Deployment failed and here is the error message. I was trying to change summary and setup to the title of the course 
![image](https://user-images.githubusercontent.com/41592024/191595532-ddef0971-a8b3-441e-8eea-e82b80d04e00.png)

  
Error in UseMethod("xml_find_all") : 
  no applicable method for 'xml_find_all' applied to an object of class "xml_document"

How to solve this problem?
Here is the full report
https://github.com/SaraMorsy/FAIR/actions/runs/3098953110/jobs/5017496308

Thank you in advance!

**Comment by SaraMorsy** [comment]
Thank you for your response! It is fixed. :partying_face:
I edited the sandpaper and it also worked. My point here for **library carpentry format**  
usually no need for setup instructions because it is more related to data management principles, not coding.

 Summary and setup can be confusing here for our audience (biologists) that's why I wanted to change it so instead **summary and setup** it would be the title of the course showing the objectives and outlines of the course
**What do you think?**

![image](https://user-images.githubusercontent.com/41592024/191918937-335191ba-6c2e-4067-a533-00961f41caba.png)

**Comment by zkamvar** [comment]
> Thank you for your response! It is fixed. :partying_face: I edited the sandpaper and it also worked.

Glad to hear it! 

> My point here for **library carpentry format** usually, no need for setup instructions because it is more related to data management principles, not coding.
> 
> Summary and setup can be confusing here for our audience (biologists) that's why I wanted to change it so instead of **summary and setup** it would be **the title of the course showing the objectives and outlines of the course** **What do you think?**

This is a good point (which also highlights one of the reasons why we are doing a beta phase of this instead of launching it unilaterally across our lessons immediately) and is also a perfect task for the future [lesson infrastructure committee](https://carpentries.org/blog/2021/12/lesson-infrastructure-committee/)! There are a couple of ways to go about this:

1. change the setup for Library Carpentry lessons to read "Objectives and Outlines"
2. allow the `title` for `index.md` to have precedence over "Summary and Setup" (e.g if you write `title: "Fair Data in Life Science"` or `title: "Objectives and Outlines"`, then it will appear on your page as you wish).
3. allow custom `titles` for both `index.md` and `learners/setup.md` so that they are combined to `[index title] and [setup title]`

The first option runs into the same problem as the original problem: the prescribed method does not fit with the cases needed (e.g. lc-git still needs a setup).

The second and third options work, but the third takes a bit more explaining, so I'm happy to go with the second. 

The reason I did not put the title of the lesson in that slot in the first place is because it is a duplicate of the pagetitle and does not clearly state what the purpose of that particular page is.

**Comment by SaraMorsy** [comment]
I agree with you, I like the second option as well, I will try it and show you the results

Thank you!

**Comment by froggleston** [comment]
Fixed in #714
